jobs/ClusterHQ_Senior_Lead_QA

```diff
@@ -0,0 +1,74 @@
+title: Senior / Lead Software Test Engineer
+company: ClusterHQ
+url: http://techfolk.co.uk/current-jobs/senior-lead-software-test-engineer-central-bristol-tl103
+location: Bristol, England
+contract: permanent
+contact:
+    name: Andrew Gifford
+    email: hello@techfolk.co.uk
+    phone: +44 (0)117 318 2447
+    ...: ...
+created: 2015-08-04
+tags:
+  - Bristol
+  - England
+  - uk
+  - relocation
+  - QA
+  - testing
+  - software testing
+  - containers 
+  - containerisation
+  - containerization
+  - virtualisation
+  - virtualization
+  - storage
+  - clustering
+  - microservices
+  - orchestration
+  - Docker
+  - Kubernetes
+  - startup
+  - open source
+
+---
+
+Your challenge
+
+ClusterHQ needs you to help establish QA for next generation cloud software, in the new domain of containerisation.
+
+Your role
+
+This is chance to be part of something new, where you can have a voice and be recognised for helping produce disruptive open source container software that will impact infrastructure, application architecture and software delivery as fundamentally in the coming years as virtualisation has in the last decade. 
+
+Joining as our first Lead QA Engineer, we welcome your ability to establish and run a robust QA function as a service, building on the foundations that are serving us well. You’ll have our full support in running good process, getting the most from our tooling and achieving best practice in TDD.
+
+Your work will pave the way for other QA colleagues to join you, each dedicated to a product engineering SCRUM team. We will look to you as we hire new team members, as we plan their work, get them up to speed and expand our operation. 
+
+ClusterHQ is offering relocation to Bristol, UK, which is fast becoming the cloud computing hub of Europe and is renowned for its high quality of life.
+
+About you
+
+•	You have established and run QA for a software product or comprehensive feature set
```

Comment:
This is unlikely to come out right in Markdown. Only '*', '+' and '-' work to indicate a bulletted list. I expect Markdown will wrap these lines together into one big paragraph with the bullets all over the place, but I may be wrong.